src/H5FDonion.h

```diff
@@ -128,9 +128,10 @@ typedef struct H5FD_onion_fapl_info_t {
 extern "C" {
 #endif
 
-H5_DLL hid_t  H5FD_onion_init(void);
-H5_DLL herr_t H5Pget_fapl_onion(hid_t fapl_id, H5FD_onion_fapl_info_t *fa_out);
-H5_DLL herr_t H5Pset_fapl_onion(hid_t fapl_id, const H5FD_onion_fapl_info_t *fa);
+H5_DLL hid_t    H5FD_onion_init(void);
+H5_DLL herr_t   H5Pget_fapl_onion(hid_t fapl_id, H5FD_onion_fapl_info_t *fa_out);
+H5_DLL herr_t   H5Pset_fapl_onion(hid_t fapl_id, const H5FD_onion_fapl_info_t *fa);
+H5_DLL hssize_t H5FDget_onion_revision_count(const char *filename, hid_t fapl_id);
```

Comment:
The last 3 functions need doxygen comments (H5FD_onion_init does not)

Reply:
I added some comments.  It's my first time to use Doxygen comment.  I hope I did the right thing.